### PR TITLE
Improved Docker Compose files detection

### PR DESCRIPTION
**Pull request by augi**

### CODE REVIEW

File: src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeUp.groovy [hunk 1]
```diff
@@ -103,14 +104,49 @@ class ComposeUp extends DefaultTask {
...
+                throw new IllegalArgumentException("Custom Docker Compose file not found: $f")
+            }
+            f
+        }
+    }
+
+    File findInParentDirectories(String filename, File directory) {
+        if ((directory) == null) return null
+        def f = new File(directory, filename)
+        f.exists() ? f : findInParentDirectories(filename, directory.parentFile)
```
Comment (alenkacz) [inline src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeUp.groovy]: to opravdu hledas od projektove slozky az do rootu? :) tak to chceme?

**Comment by alenkacz** [comment]
testy nad compose 1.6 mame? pokud jo, pak :+1:

**Comment by augi** [comment]
We are using 1.6.2 :)

File: src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeUp.groovy (hunk 1 above)
Comment (augi) [inline src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeUp.groovy]: Yes, it's how docker-compose is looking for the YML file and must do the same here.

**Pull request merged**